courses/machine_learning/deepdive2/launching_into_ml/solutions/improve_data_quality.ipynb

```diff
@@ -34,36 +34,10 @@
    "metadata": {},
```

Comment:
<p>Replace line 1, with this:  In Python&nbsp;shape()&nbsp;is used in pandas to give the number of rows/columns.  The number of rows is given by .shape[0].  The number of columns is given by .shape[1].  Thus,  <code>shape()</code>&nbsp;consists of an array having two arguments -- rows and columns</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1267/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Updated the same.